test/commands/exercise/representation/create_search_index_document_test.rb

```diff
@@ -44,83 +44,56 @@ class Exercise::Representation::CreateSearchIndexDocumentTest < ActiveSupport::T
   test "indexes representation" do
     content = "CONTENT!!"
     num_published_solutions = 20
+    reputation = 1234
+    num_loc = 42
+
+    solutions = Array.new(2).map do
+      create(:practice_solution, :published, published_iteration_head_tests_status: :passed).tap do |solution|
+        submission = create(:submission, solution:)
+        create(:iteration, submission:)
+      end
+    end
```

Comment:
Maybe create a non-published solutions before this code block to sanity check that only published solutions are considered?